interbotix_ros_xsarms/examples/interbotix_xsarm_gravity_compensation/config/mode_configs.yaml

```
@@ -0,0 +1,21 @@
+# This file is used to customize the initial operating modes of all joint groups and individual joints in a robot.
+# It should be placed in the 'config' directory of any ROS package that builds ontop of the interbotix_xs_sdk.
+# Example values are shown as well. All parameters are mandatory unless stated otherwise
+
+port: /dev/ttyDXL_awx250s                             # Optional; specifies the USB port that connects to the U2D2;
```

Keep this as `/dev/ttyDXL`

```suggestion
port: /dev/ttyDXL                             # Optional; specifies the USB port that connects to the U2D2;
```

---

Ah, sorry I missed this. I was testing on the arm and forgot to change it back.